### PR TITLE
[ADD] module to restore placeholder in partner_id field

### PR DESCRIPTION
**Pull request by kaerdsar**
* [ADD] module to restore placeholder in partner_id field of reconciliation widget.
* [IMP] update account_bank_statement_import_qif to show partner name even if the partner is not found.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please don't use plurals in names. I suggest ` account_reconciliation_widget_partner` or ` account_reconciliation_partner_placeholder`. Any way, AFAIK, there's already another placeholder in the partner combo. Can't you use that one?

**Comment by kaerdsar** [comment]
@pedrobaeza Thanks for the quick check. I like account_reconciliation_partner_placeholder. Don't know about the other placeholder, but I can try to find it. Is it in other OCA module?

**Comment by pedrobaeza** [comment]
I'm talking about standard, but I haven't seen yet in v11. In v10, I find a placeholder in the partner combo box:

seleccion_006

**Comment by kaerdsar** [comment]
@feketemihai could you please take a look?

**Comment by kaerdsar** [comment]
@pedrobaeza I see the combo box empty in the image you attached. Where did you find the placeholder?

**Comment by pedrobaeza** [comment]
I know, I have taken directly a runbot v11 screenshot, but with no possibility of changing data for showing it. As said, I have seen it in v10, but not v11, so maybe it's a removed feature.

**Comment by kaerdsar** [comment]
Ok, I will also check in v10 in order to compare. Thanks

**Comment by feketemihai** [comment]
@pedrobaeza The ideea of the module is to have another combo in case of not matching partner... Which is done by account number,  it's a feature from v8 where Odoo made a assumption of the partner based also on partner name from imported dict... And clicking on the combo will select the partner.

**Comment by feketemihai** [comment]
And it will only show when no partner matching, otherwise will not see any change.

**Comment by pedrobaeza** [comment]
OK, thanks for the clarification.

**Comment by feketemihai** [comment]
@pedrobaeza This is how it looks with the placeholder...
placeholder
Please see difference between partners found and the one that have partner name only

**Comment by feketemihai** [comment]
@kaerdsar Can you please rename the module, not much to review since it's just a small improvement and it's working ok.

**Comment by kaerdsar** [comment]
@feketemihai I already renamed the module to 'account_reconciliation_partner_placeholder'. Do you have any other suggestion for the name?

**Review by feketemihai — APPROVED** [review]

File: account_reconciliation_partner_placeholder/static/src/js/reconciliation/reconciliation_renderer.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+odoo.define('lr_account.ReconciliationRenderer', function (require) {
```
Comment (StefanRijnhart) [inline account_reconciliation_partner_placeholder/static/src/js/reconciliation/reconciliation_renderer.js:1]: Would it be good convention to use the full module name for the prefix?

Comment (pedrobaeza) [reply account_reconciliation_partner_placeholder/static/src/js/reconciliation/reconciliation_renderer.js:1]: Yeah, I think so. We were using that as a 'de facto' convention.

**Comment by pedrobaeza** [comment]
There's an opened PR in Odoo for restoring it in core, not via a module: https://github.com/odoo/odoo/pull/26172

**Comment by pedrobaeza** [comment]
Closing as now there's no need of this thanks to odoo/odoo#26172

**Pull request closed without merging**